ntfs_fare_extension_fr_deprecated.md

```diff
@@ -131,3 +137,8 @@ Avant d'appliquer le ticket, on va donc vérifier la validité de :
 * la condition sur le debut de la section à emprunter (zone tarifaire, zone d'arrêt, durée, nombre de changements, ticket)
 * la condition sur la fin de la section à emprunter (zone tarifaire, zone d'arrêt, durée)
 * la condition globale
+
+NB: les identifiants des objets spécifiés dans les conditions de début et fin de 
```

Comment:
To me, having the same ids (including case) than NTFS would be nicer. As this lowercasing is done for now, but it may not last? (well... will v1 last? :smile: )

Also if we keep that recommendation, you may update the example added :wink: (as well as a lot of the examples, but...).

Reply:
You're maybe right, I'll delete the recommendation. However, the note should be kept I think.